### PR TITLE
boards: arm: sensortile_box_pro: init CDC_ACM console only if present.

### PR DESCRIPTION
**Pull request by msmttchr**
Avoid conflict with applications which don’t require a console.

### CODE REVIEW

**Review by avisconti — DISMISSED** [review]
LGTM.

**Comment by erwango** [comment]
Can't be merged before HWMV2 branch is merged (eow)

File: boards/arm/sensortile_box_pro/board.c
Comment (erwango) [inline boards/arm/sensortile_box_pro/board.c:14]: I'm not sure to understand the dependency with `CONFIG_UART_CONSOLE` here.
Isn't `CONFIG_CONSOLE` sufficient ?

Comment (msmttchr) [reply boards/arm/sensortile_box_pro/board.c:14]: @erwango, in my understanding, `zephyr_cdc_acm_uart` is a USB implementation for UART console.
I could imagine situation where user wants to have a console, but not over UART, eg. (RAM_CONSOLE), in this case, I would not want to enable the `zephyr_cdc_acm_uart`.

Comment (erwango) [reply boards/arm/sensortile_box_pro/board.c:14]: > in my understanding, zephyr_cdc_acm_uart is a USB implementation for UART console.

Exact! I forgot about it. It is then perfectly fine indeed.

**Review by erwango — DISMISSED** [review]

**Review by nordicjm — DISMISSED** [review]
Needs converting to hwmv2

**Comment by msmttchr** [comment]
> Needs converting to hwmv2

@nordicjm, you mean we need to wait for hwmv2 to merge and then modify/rebase this PR ?

**Comment by nordicjm** [comment]
hwmv2 is merged, this PR needs to be rebased and updated to use it

**Comment by msmttchr** [comment]
> hwmv2 is merged, this PR needs to be rebased and updated to use it

Done. Thanks

**Review by avisconti — APPROVED** [review]

**Review by gautierg-st — APPROVED** [review]

**Comment by aescolar** [comment]
@erwango your approval as assignee is missing :)

**Review by erwango — APPROVED** [review]

**Pull request merged**
